ribs/emitters/operators/_gaussian.py

```diff
@@ -0,0 +1,48 @@
+"""Gaussian Operator"""
+import numpy as np
+
+from ribs.emitters.operators._operator_base import OperatorBase
+
+
+class GaussianOperator(OperatorBase):
+    """Args:
```

Comment:
Can you add a one-line description at the beginning? And maybe a short paragraph if needed. See the Google-style docstrings I mentioned for more info.